docs/user/discover.asciidoc

```diff
@@ -3,9 +3,30 @@
 
 [partintro]
 --
-**_Tell {kib} where to find your data, then search and filter it for hidden insights and relationships._**
+**_Exploring and querying your data._**
+
+*Discover* is your window into your data in {es}. *Discover* enables you to 
+quickly search and filter your data, get information about structure of the fields, 
+and navigate to *Visualize* and *Maps* to present your data.
+You can also customize and save your searches and place them on a dashboard.
+
+++++
+<script type="text/javascript" async src="https://play.vidyard.com/embed/v4.js">
+</script>
+<img
+style="width: 100%; margin: auto; display: block;"
+class="vidyard-player-embed"
+src="https://play.vidyard.com/c2qBPz74BU16ncbvDWGvbz.jpg"
+data-uuid="c2qBPz74BU16ncbvDWGvbz"
+data-v="4"
+data-type="inline"
+/>
+++++
 
-You’ve added your data, and now you’re ready to dig in. You have questions about your data.
+[float]
+=== Select, explore, and present your data
```

Comment:
This title fell flat for me.  What do you think about removing the tag and using "Exploring and querying your data" as the title here instead.

Reply:
I played around with it, was not 100% sure what's the tag you mean and where the title should be in the end. When I arranged stuff a little, rewriting a bit, I came to the following result? which I find somehow "geschmeidig", what do you think? 

<img width="800" alt="Bildschirmfoto 2021-03-04 um 14 52 02" src="https://user-images.githubusercontent.com/463851/109974197-97c67a00-7cf9-11eb-80a5-3a41e378363e.png">